.github/workflows/style.yml

```diff
@@ -15,7 +15,7 @@ permissions:
   contents: read
 
 jobs:
-  build:
+  style:
```

Comment:
While this change is good --- the job to have a unique id (i.e., `needs: style`) --- it should be committed separately.